delete_runs.sh

```
@@ -0,0 +1,17 @@
+gh api --paginate /repos/parsely/wp-parsely/actions/runs | \
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)